traffic_ops/app/templates/navbar.html.ep

```diff
@@ -14,6 +14,7 @@
  -->
 
 <!-- Note: add a page here, mod the site.css as well! -->
+<div class="deprecatedBar">WARNING: This old interface is deprecated! It will be removed in Traffic Control 4.0! Please migrate to <a href="<%= $config->{'portal'}{'external_url'} %>"> Traffic Portal</a> soon!</div>
```

Comment:
Do you really want to commit to a release? how about just:

WARNING: This old interface is deprecated! Functionality may or may not work as expected. If installed, please use the <a href="<%= $config->{'portal'}{'base_url'} %>"> Traffic Portal</a> (<a href="<%= $config->{'portal'}{'external_url'} %>">Traffic Portal Docs</a>).

Reply:
Yes, we really want to commit to a release. It has taken extensive negotiating to get agreement to deprecate and remove. We are deprecating in 3.0 and removing in 4.0. The banner should explicitly state that. There should be no ambiguity, or argument that the deprecation deadline was unclear.